src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -128,37 +154,66 @@ public ReadOnlyEntryBook getListEntryBook() {
     }
 
     @Override
-    public boolean hasEntry(Entry entry) {
-        requireNonNull(entry);
-        return listEntryBook.hasPerson(entry);
+    public boolean hasListEntry(Entry listEntry) {
+        requireNonNull(listEntry);
+        return listEntryBook.hasPerson(listEntry);
     }
 
     @Override
-    public void deleteEntry(Entry target) {
+    public void deleteListEntry(Entry target) {
         listEntryBook.removePerson(target);
     }
 
     @Override
-    public void addEntry(Entry entry) {
+    public void addListEntry(Entry entry) {
         listEntryBook.addEntry(entry);
         updateFilteredEntryList(PREDICATE_SHOW_ALL_ENTRIES);
     }
 
     @Override
-    public void setEntry(Entry target, Entry editedEntry) {
+    public void setListEntry(Entry target, Entry editedEntry) {
         requireAllNonNull(target, editedEntry);
 
         listEntryBook.setPerson(target, editedEntry);
     }
 
     @Override
-    public void clearEntryBook() {
+    public void clearListEntryBook() {
         listEntryBook.clear();
     }
 
+    //=========== Archives EntryBook ================================================================================
+
     @Override
-    public void displayEntryBook(ReadOnlyEntryBook entryBook) {
-        displayedEntryList.set(entryBook.getEntryList());
+    public void setArchivesEntryBook(ReadOnlyEntryBook archivesEntryBook) {
+        this.archivesEntryBook.resetData(archivesEntryBook);
+    }
+
+    @Override
+    public ReadOnlyEntryBook getArchivesEntryBook() {
+        return archivesEntryBook;
+    }
+
+    @Override
+    public boolean hasArchivesEntry(Entry archiveEntry) {
+        requireNonNull(archiveEntry);
+        return archivesEntryBook.hasPerson(archiveEntry);
+    }
+
+    @Override
+    public void deleteArchivesEntry(Entry target) {
```

Comment:
These functions are not used outside `ModelManager` so should we still make them available in `Model`?

Reply:
Used now